Examples/Scripts/Python/geant4_parallel.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from multiprocessing import Pool
+from functools import partial
+
+import numpy as np
+
+# This script runs a Geant4 simulation in parallel by passing chunks of events to subprocesses.
+# This is a workaround to achieve parallel processing even though Geant4 is not thread-save
+# and thus the internal parallelism of the ACTS examples framework cannot be used.
+#
+# Note that:
+# * This should give equivalent results to a sequential run if the RNG is initialized with
+#   the same seed in all runs
+# * This works only for csv outputs, since they write the results in one file per event
+#   (the naming of the csv-files should be equivalent to a sequential run)
+
+
+def runGeant4EventRange(events, outputDir):
+    import acts
+    import acts.examples
+    from acts.examples.simulation import addParticleGun, addGeant4, EtaConfig
+    from acts.examples.odd import getOpenDataDetector
+    from common import getOpenDataDetectorDirectory
+
+    u = acts.UnitConstants
+
+    detector, trackingGeometry, decorators = getOpenDataDetector(
+        getOpenDataDetectorDirectory()
+    )
+
+    field = acts.ConstantBField(acts.Vector3(0, 0, 2 * u.T))
+    rnd = acts.examples.RandomNumbers(seed=42)
+
+    s = acts.examples.Sequencer(events=len(events), skip=events[0], numThreads=1)
+
+    outputDir = Path(outputDir)
+    addParticleGun(
+        s,
+        EtaConfig(-2.0, 2.0),
+        rnd=rnd,
+        outputDirCsv=outputDir / "csv",
+        outputDirRoot=None,
+    )
+    addGeant4(
+        s,
+        detector,
+        trackingGeometry,
+        field,
+        outputDirCsv=outputDir / "csv",
```

Comment:
Do we have to do CSV here?

Reply:
what do you mean? we cannot do ROOT output, so if we want to get something out of this we need to do CSV?

Reply:
Why does ROOT not work?

Reply:
Because then we have multiple root writers in the different processes that open the same file and try to write to it. I'm pretty sure that does not work out well...

Reply:
But can't we do multiple files and `hadd` afterwards?

Reply:
Yeah in principle yes, of course. But for my purpose I need the csv-files anyways and I actually don't want to implement this right now (and it's not possible without thirdparty libraries I guess).
The nice thing about the CSV output is that it gives equivalent output kind of "out of the box", so I wanted to share it...

I could add a line in the comments that indicates how one could achieve this with root output as well

Reply:
Ok. The only reason I bring this up is that the CSV files are ludicrously large. `hadd` is part of ROOT.

But ok to go ahead with your suggestion, just asking.